flink-runtime/src/main/java/org/apache/flink/runtime/net/SSLUtils.java

```diff
@@ -375,12 +435,36 @@ private static SslContext createRestNettySSLContext(
     public static SslContext createRestNettySSLContext(
```

Comment:
note: this is unused, but I left it here since it's public.  Do we care about removing it and changing the public interface here?